letsencrypt/acme/messages2_test.py

```diff
@@ -75,6 +76,12 @@ def test_repr(self):
         self.assertEqual('MockConstant(a)', repr(self.const_a))
         self.assertEqual('MockConstant(b)', repr(self.const_b))
 
+    def test_equality(self):
```

Comment:
Used assertBool with == and != for clarity as to which operator was actually being called